contributing/setup.md

```diff
@@ -17,7 +17,7 @@ If you're contributing to pachyderm, you may need the following additional setup
 
 ## General Requirements
 
-Follow the [general setup instructions](https://github.com/pachyderm/pachyderm/blob/master/SETUP) to install:
+Install:
```

Comment:
Previous link to https://github.com/pachyderm/pachyderm/blob/master/SETUP is dead. Couldn't find a similar looking "SETUP" file so deleting it all together